### ISSUE TITLE
Needs theme with no background color

### ISSUE THREAD

**Issue by Cretezy** [banga/git-split-diffs#7]
I use a black background in my terminal, and the "dark" themes aren't really enough. Either having a black theme or one without background color (meaning using the terminal's background) would be ideal.

Great project!

**Comment by controversial** [comment]
Second this. I use a terminal with a translucent background:

<img width="728" alt="Screen Shot 2021-05-01 at 8 26 09 PM" src="https://user-images.githubusercontent.com/10377391/116798228-798fb700-aabb-11eb-9907-c2bd9b26949f.png">

however, git-split-diffs adds an opaque background behind every character on the screen:

<img width="728" alt="Screen Shot 2021-05-01 at 8 28 28 PM" src="https://user-images.githubusercontent.com/10377391/116798261-cc696e80-aabb-11eb-9b23-8e5c08e5885e.png">

It would be great if there were an option to render a theme without any background color on the non-highlighted lines (obviously, the lines highlighted as added or removed need to have a background color)

**Comment by banga** [comment]
Makes sense. The reason everything has a background is because I added support for alpha in colors, so that you could say e.g. line deletions are `ff000022` which adds a reddish tinge to the background. This is especially useful if you have multiple layers like for highlighting inline changes.

Anyway, will think of a fix. Would you rather have a way to unset background colors in existing themes, or just have a separate theme without backgrounds? I'm guessing the former would be better so you can use any theme

**Comment by Cretezy** [comment]
Alternatively, a way to point to a custom `.json` theme file, so everyone can be happy 😃

**Comment by banga** [comment]
Oh yeah that might be better. You can  hackishly do that right now by making `theme-name` a relative path, but there should be a better way

**Comment by controversial** [comment]
I hacked together a quick fix in #8!

**Comment by banga** [comment]
I just published v2.2.0 which adds support for using custom themes. The instructions are at https://github.com/banga/git-split-diffs/tree/v2.2.0?tab=readme-ov-file#custom-themes and https://github.com/banga/git-split-diffs/tree/v2.2.0?tab=readme-ov-file#want-to-remove-background-colors-from-a-theme. So far from personal usage, just removing `backgroundColor` seems to be enough, but let me know what you all think.

**Comment by Yukaii** [comment]
I'm getting the following error when  `"SYNTAX_HIGHLIGHTING_THEME": "my-custom-theme-name",` is set in my custom theme JSON. I currently use theme names from [shikijs](https://shiki.style/themes) to workaround this issue

```
ShikiError: Theme `my-custom-theme-name` is not included in this bundle. You may want to load it from external source.
```

**Comment by banga** [comment]
Yeah to be clear, I've only added support to use custom themes for `git-split-diffs`. Syntax highlighting happens via shikijs which has its own themes. I'll look into it though, should be easy to add. It would actually be nice to have a way to just use a shikijs theme and derive all the colors from it for `git-split-diffs`.

**Comment by danielwerg** [comment]
transparent theme I've just made:

```json
{
  "SYNTAX_HIGHLIGHTING_THEME": "github-dark-dimmed",
  "DEFAULT_COLOR": {
    "color": "#768390",
    "backgroundColor": "#00000000"
  },
  "COMMIT_HEADER_COLOR": {
    "color": "#adbac7",
    "backgroundColor": "#00000000"
  },
  "COMMIT_HEADER_LABEL_COLOR": {},
  "COMMIT_SHA_COLOR": {},
  "COMMIT_DATE_COLOR": {},
  "COMMIT_AUTHOR_COLOR": {},
  "COMMIT_MESSAGE_COLOR": {
    "color": "#909dab",
    "backgroundColor": "#00000000"
  },
  "COMMIT_TITLE_COLOR": {
    "color": "#c6e6ff",
    "modifiers": ["bold"]
  },
  "BORDER_COLOR": {
    "color": "#444c56",
    "backgroundColor": "#00000000"
  },
  "FILE_NAME_COLOR": {
    "color": "#adbac7",
    "backgroundColor": "#00000000"
  },
  "HUNK_HEADER_COLOR": {
    "backgroundColor": "#00000000"
  },
  "DELETED_WORD_COLOR": {
    "backgroundColor": "#c93c3733"
  },
  "INSERTED_WORD_COLOR": {
    "backgroundColor": "#46954a33"
  },
  "DELETED_LINE_NO_COLOR": {
    "color": "#e5534b",
    "backgroundColor": "#c93c371a"
  },
  "INSERTED_LINE_NO_COLOR": {
    "color": "#57ab5a",
    "backgroundColor": "#46954a1a"
  },
  "UNMODIFIED_LINE_NO_COLOR": {},
  "DELETED_LINE_COLOR": {
    "backgroundColor": "#442d3080"
  },
  "INSERTED_LINE_COLOR": {
    "backgroundColor": "#2a3c3380"
  },
  "UNMODIFIED_LINE_COLOR": {},
  "MISSING_LINE_COLOR": {
    "backgroundColor": "#2d333b80"
  }
}
```

**Comment by lukejans** [comment]
Is it possible to combine terminal base-16 colors with a theme? I usually use dark and light modes in my terminal, and I’d love a way to set a theme that uses my terminal colors while still allowing me to access colors with opacity for the actual differences.
